File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/DomainOrderTldServices.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Example map of TLDs to service IDs for domain ordering.
 */
class DomainOrderTldServices
{
    /**
     * Service ID for .asia TLD
     */
    #[DTA\Data(field: ".asia", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $_asia = null;

    /**
     * Service ID for .be TLD
     */
    #[DTA\Data(field: ".be", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $_be = null;

    /**
     * Service ID for .biz TLD
     */
    #[DTA\Data(field: ".biz", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $_biz = null;

    /**
     * Service ID for .ca TLD
     */
    #[DTA\Data(field: ".ca", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $_ca = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/DomainOrderTldServices.scala
package io.swagger.server.model


/**
 * Example map of TLDs to service IDs for domain ordering.
 *
 * @param .asia Service ID for .asia TLD for example: ''10001''
 * @param .be Service ID for .be TLD for example: ''10002''
 * @param .biz Service ID for .biz TLD for example: ''10003''
 * @param .ca Service ID for .ca TLD for example: ''10004''
 */
case class DomainOrderTldServices (
  .asia: Option[Int],
  .be: Option[Int],
  .biz: Option[Int],
  .ca: Option[Int]
)

